test_automl/sweep-config.yaml

```diff
@@ -0,0 +1,21 @@
+job: xzy11632/pytorch-cell_type_annotation_ACTINN_function_new/job-source-pytorch-cell_type_annotation_ACTINN_function_new-test_automl_test_automl_fun_job.py:latest
+metric:
+  goal: maximize
+  name: scores
+run_cap: 20
+scheduler:
+  job: xzy11632/pytorch-cell_type_annotation_ACTINN_function_new/job-OptunaScheduler:latest
+  num_workers: 2
+  docker_image: rdance_fcf2b32f.sif
```

Comment:
A slight change has been made to the source code of WanDB, and the SIF file of Singularity can be used